images/cdi-artifact/patches/012-add-caps-for-deckhouse-provisioners.patch

```diff
@@ -1,26 +1,14 @@
 diff --git a/pkg/storagecapabilities/storagecapabilities.go b/pkg/storagecapabilities/storagecapabilities.go
-index 24bbac64b..fccdd0c08 100644
+index 66cd33025..74349a9cf 100644
 --- a/pkg/storagecapabilities/storagecapabilities.go
 +++ b/pkg/storagecapabilities/storagecapabilities.go
-@@ -8,9 +8,10 @@ import (
- 
- 	v1 "k8s.io/api/core/v1"
- 	storagev1 "k8s.io/api/storage/v1"
--	"kubevirt.io/containerized-data-importer/pkg/util"
- 	"sigs.k8s.io/controller-runtime/pkg/client"
- 
-+	"kubevirt.io/containerized-data-importer/pkg/util"
-+
- 	cdiv1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
- )
- 
-@@ -102,6 +103,9 @@ var CapabilitiesByProvisionerKey = map[string][]StorageCapabilities{
- 	"manila.csi.openstack.org": {{rwx, file}},
- 	// ovirt csi
- 	"csi.ovirt.org": createRWOBlockAndFilesystemCapabilities(),
+@@ -50,6 +50,9 @@ var CapabilitiesByProvisionerKey = map[string][]StorageCapabilities{
+ 	"openshift-storage.cephfs.csi.ceph.com": {{rwx, file}},
+ 	// LINSTOR
+ 	"linstor.csi.linbit.com": createAllButRWXFileCapabilities(),
 +	// Deckhouse
-+	"replicated.csi.storage.deckhouse.io": createLinstorCapabilities(),
++	"replicated.csi.storage.deckhouse.io": createAllButRWXFileCapabilities(),
 +	"local.csi.storage.deckhouse.io":      createTopoLVMCapabilities(),
```

Comment:
Our block is moved to be closer to LINSTOR to quicker understand what's changed in upstream in the future.